moveit_ros/moveit_servo/include/moveit_servo/servo_calcs.h

```diff
@@ -314,14 +305,12 @@ class ServoCalcs
   // Main tracking / result publisher loop
   std::thread thread_;
   bool stop_requested_;
-  std::atomic<bool> done_stopping_;
```

Comment:
This variable was only used when incoming velocity commands were all zero. We don't bother tracking that anymore.